src/test/java/org/folio/ld/dictionary/model/ResourceTest.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.ld.dictionary.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.folio.ld.dictionary.PredicateDictionary;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.junit.jupiter.api.Test;
+
+class ResourceTest {
+
+  @Test
+  void addType_shouldAddType() {
+    Resource resource = new Resource();
```

Comment:
minor: we can use ```var``` here and in the methods below.

Reply:
done